packages/devtools_app/lib/src/shared/server/_analytics_api.dart

```diff
@@ -28,7 +32,8 @@ Future<bool> isAnalyticsEnabled() async {
   if (isDevToolsServerAvailable) {
```

Comment:
We probably want to fix the dartdoc here and not mention the legacy file and speak generally

Reply:
Changed things around a bit so we don't interact with the DTD unified analytics server at all from the DevTools server wrapper. These server APIs are now exclusively for the legacy analytics.

Reply:
ahh i see now, that makes sense, that's a clean solution!